rust/src/arrow/linalg/svd.rs

```diff
@@ -0,0 +1,229 @@
+use std::sync::Arc;
+
+use super::matrix::MatrixView;
+use arrow_array::Float32Array;
+
+#[allow(unused_imports)]
+#[cfg(target_os = "macos")]
+use accelerate_src;
+
+#[allow(unused_imports)]
+#[cfg(all(any(target_os = "linux", target_os = "windows"), not(docsrs)))]
```

Comment:
The `docsrs` flag is now redundant.

```suggestion
#[cfg(any(target_os = "linux", target_os = "windows")))]
```